Link to extension and proposed file with requirements (#65)
OpenRefine/sample-extension: README.md

See [our guide to writing extensions](https://openrefine.org/docs/technical-reference/writing-extensions) for more information about the process.

### Getting started
Make sure to familiarize yourself with [our extension developer guidelines](https://openrefine.org/docs/technical-reference/extension-dev-guidelines).

To start your own extension from this template, click the "Use this template" button in the top right corner of this page.

> This will create a copy of this repository, where you can then change:

Reviewer: ```suggestion
This will create a copy of this repository, where you can then change:
* The license declaration for the repo (if you want a different license)
```

* The extension name and description in `module/MOD-INF/module.properties`
* The `groupId`, `artifactId`, `name` and `description` fields in `pom.xml`
* Edit this `README.md` file to describe your extension to potential users and contributors instead of the sample extension's own instructions

### General Release 

> Please ensure that your repository contains the general release file with the relevant information.

Reviewer: ```suggestion
Please update the [GeneralRelease.md](GeneralRelease.md) file with the relevant information.
```
